draft-ietf-quic-transport.md

```diff
@@ -3747,6 +3747,20 @@ which could prevent the connection from ever becoming idle.  Non-ack-eliciting
 packets are eventually acknowledged when the endpoint sends an ACK frame in
 response to other events.
 
+An endpoint that is only sending ACK frames will not receive acknowledgments
+from its peer unless those acknowledgments are included in packets with
+ack-eliciting frames.  An endpoint SHOULD send an ACK frame with other frames
+when there are new ack-eliciting packets to acknowledge.  When only
+non-ack-eliciting packets need to be acknowledged, an endpoint MAY wait until an
+ack-eliciting packet has been received to include an ACK frame with outgoing
+frames.
```

Comment:
This is pre-existing, but it always takes me a few tries understand the intent of this MAY, so here's a small suggestion which I believe clarifies it.

```suggestion
when there are new ack-eliciting packets to acknowledge.  When only
non-ack-eliciting packets need to be acknowledged, an endpoint MAY
choose to not send an ACK frame with outgoing frames until an
ack-eliciting packet has been received.
```